df.entities.xml

```diff
@@ -2,22 +2,22 @@
     <struct-type type-name='entity_occasion_info'>
         <stl-vector name="occasions" pointer-type='entity_occasion' since='v0.42.01'/>
         <int32_t name="next_occasion_id"/>
-        <static-array type-name='int32_t' count='10'/>
-        <int32_t />
+        <static-array name='events' type-name='int32_t' count='10' ref-target='history_event'/>
+        <int32_t name='count' comment="number of elements used in array above"/>
     </struct-type>
-    <struct-type type-name='entity_occasion'>
+    <struct-type type-name='entity_occasion' comment="some festivals are annual, some single time. anon_1=0 plus anon_3=0 seems to match with single time, which doesn't make much sense. Only frequency seen is yearly">
```

Comment:
It would be better to give these fields `unk` names to make them consistent (I can address)

Reply:
I'm happy to do so, as I dislike anons. I've generally refrained from doing it as someone made them anons originally.

Reply:
It's really just the lack of a `name` attribute that makes them anon fields, so whoever put these in probably found it easier to (for example) copy `<int32_t/>` 5 times and leave it. Not particularly important to preserve anon names since they're unstable anyway.